Sources/BraintreeUIComponents/PaymentButtonView.swift

```diff
@@ -38,6 +40,9 @@ struct PaymentButtonView<Color: PaymentButtonColorProtocol>: View {
                     lineWidth: color.hasOutline ? 1 : 0
                 )
         )
+        .onLongPressGesture(minimumDuration: 0, maximumDistance: .infinity, pressing: { pressing in
```

Comment:
It looks like this method signature may have changed slightly: https://developer.apple.com/documentation/swiftui/view/onlongpressgesture(minimumduration:maximumdistance:perform:onpressingchanged:)